automatic/git.portable/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,32 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+$filePath32 = "$toolsDir\PortableGit-2.11.0-32-bit.7z.exe"
+$filePath64 = "$toolsDir\PortableGit-2.11.0-64-bit.7z.exe"
+$filePath = if (Get-ProcessorBits -eq 64) { $filePath64 } else { $filePath32 }
```

Comment:
Done. Also added the same logic for the git.install package: https://github.com/chocolatey/chocolatey-coreteampackages/pull/489/files#diff-38b497b16f7f35551e33f1702ceffb25R69